tinymce/tinymce
# Use aria-controls instead of aria-owns in DropdownUtils

**Description of Changes:**
Don't set `aria-owns` in DropdownUtils.

This fixes the issues with assistive technologies mentioned in #7300.

Test Plan:
* Verify #7300 is resolved
* Verify each of the following continues to work properly with keyboard navigation and screen readers:
   - Dropdowns
   - SplitDropdowns

**Pre-checks:**
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)

**Review:**
* [x] Milestone set
* [x] Review comments resolved

**GitHub issues**: fixes #7300
